EIPS/eip-draft_disable_selfdestruct.md

```diff
@@ -0,0 +1,63 @@
+---
+eip: <to be assigned>
+title: Disable SELFDESTRUCT
+author: Alexey Akhunov (@AlexeyAkhunov)
+discussions-to: https://ethereum-magicians.org/t/eip-for-disabling-selfdestruct-opcode/4382
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-06-25
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+Disable SELFDESTRUCT
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+Make `SELFDESTRUCT` EVM operation no-op (effectively disable). Contracts will not be able to delete themselves.
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+Althouth `SELFDESTRUCT` originally came with EVM to help clean up the state, we learnt that in practice it did not achieve this objective on
```

Comment:
```suggestion
Although `SELFDESTRUCT` originally came with EVM to help clean up the state, we learnt that in practice it did not achieve this objective on
```

Reply:
Ooh, how did you do this @lightclient?

Reply:
There should be a plus/minus button as part of the comment interface :)